src/foam/nanos/approval/services.jrl

```diff
@@ -46,7 +46,7 @@ p({
         .build())
       .build()
       .addPropertyIndex(new foam.core.PropertyInfo[] { foam.nanos.approval.ApprovalRequest.CREATED })
-      .addPropertyIndex(new foam.core.PropertyInfo[] { foam.nanos.approval.ApprovalRequest.DAO_KEY })
+      .addPropertyIndex(new foam.core.PropertyInfo[] { foam.nanos.approval.ApprovalRequest.DAO_KEY, foam.nanos.approval.ApprovalRequest.CLASSIFICATION })
```

Comment:
I recommend two indexes,
1) DAO_KEY on it's own
2) the pair you've just added.